_appmap/test/test_django.py

```diff
@@ -171,9 +171,6 @@ def raise_on_call(*args):
 
     assert events[1].event == "return"
     assert events[1].parent_id == events[0].id
-    assert events[1].exceptions == [
```

Comment:
After the conversation in Slack, I realized that I'd misread the appmap spec: `http_server_response` events aren't supposed to have an `exceptions` property (at least, I don't think they are).

In reality, it doesn't seem like there are many (maybe no) cases where this would add any information. Exceptions that get raised in application code will show up in the `return` events for functions, and the status in the `http_server_response` will accurately reflect the response that django sends when an unhandled exception gets raised.